utils/constants.ts

```
@@ -1,5 +1,15 @@
 import { useAuthenticationStore } from '@/store/uistate/features/authentication';
 
+export const CRM_URL = process.env.NEXT_PUBLIC_CRM_URL || process.env.CRM_URL;
+export const CRM_API_URL = CRM_URL;
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)